test/basic-querying.test.js

```diff
@@ -57,8 +60,10 @@ describe('basic-querying', function() {
       User.destroyAll(done);
     });
 
+    var id = isCassandraConnector ? uuidV4() : 1;
```

Comment:
implemented generateId and removed all the if isCassandraConnector ? uuidV4.  removed uuidV4 dependency as well.  Mongo connector uses ObjectID.  Cassandra connector, i'd like to keep Uuid.